OMCompiler/SimulationRuntime/c/simulation/solver/gbode_main.c

```diff
@@ -1026,6 +1087,8 @@ int gbodef_main(DATA *data, threadData_t *threadData, SOLVER_INFO *solverInfo, d
     memcpy(gbfData->yv, gbfData->yRight, nStates * sizeof(double));
     memcpy(gbfData->kv, gbfData->kRight, nStates * sizeof(double));
 
+    debugRingBufferSteps(LOG_GBODE, gbfData->yv, gbfData->kv, gbfData->tv, nStates,  gbfData->ringBufferSize);
+
```

Comment:
Yes, I would prefer to keep them. This stored information is also used for extrapolation in case of implicit RK methods and I would like to be able to monitor this.